### ISSUE TITLE
Searxng should implement website ratings

### ISSUE THREAD

**Issue by davidcollini** [searxng/searxng#3228]
**Is your feature request related to a problem? Please describe.**

[Ecosia](https://ecosia.helpscoutdocs.com/article/393-green-features) and [Duckduckgo](https://spreadprivacy.com/privacy-simplified/) have Environmental and Privacy ratings, and I would love to see Searxng implement this feature into its results aswell

**Describe the solution you'd like**
I would like to have a setting where Searxng would add a privacy, environmental, and moral rating that follows the ratings from these sites or other similar sites

[Ecosia](https://ecosia.helpscoutdocs.com/article/393-green-features)

[Duckduckgo](https://spreadprivacy.com/privacy-simplified/)

[Goodonyou.eco](https://goodonyou.eco/)

**Describe alternatives you've considered**
This feature should probably be off by default, and Searxng instances should be allowed to enable them

**Additional context**

I opened the same issue for [Stract](https://github.com/StractOrg/stract) in [here](https://github.com/StractOrg/stract/issues/154#issuecomment-1950261316), if you want to work with @mikkeldenker on the implementation

**Comment by return42** [comment]
FWI there is a [only_show_green_results](https://docs.searxng.org/dev/plugins.html#external-plugins) plugin / Its my implementation of https://github.com/searx/searx/pull/1878 ([see commits return42/tgwf-searx-plugins/](https://github.com/return42/tgwf-searx-plugins/commits/master/)).. never heard that some has used it .. that's why I stooped further investigations.

**Comment by davidcollini** [comment]
> FWI there is a [only_show_green_results](https://docs.searxng.org/dev/plugins.html#external-plugins) plugin / Its my implementation of https://github.com/searx/searx/pull/1878 ([see commits return42/tgwf-searx-plugins/](https://github.com/return42/tgwf-searx-plugins/commits/master/)).. never heard that some has used it .. that's why I stooped further investigations.

I don't believe that we should limit results shown to the user, I think we should just offer additional information to users who want it

Additionally, if the user wants to limit their results, they should be able to

**Comment by secondtruth** [comment]
Good idea!

**Comment by return42** [comment]
> I would like to have a setting where Searxng would add a privacy, environmental, and moral rating that follows the ratings from these sites or other similar sites.

sorry, not planned
